### PR TITLE
[Keyboard] Add Z60 Solder

### PR DESCRIPTION
**Pull request by tommyamoszhao**
## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/quarkeys/z60/config.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#define RGBLIGHT_EFFECT_ALTERNATING
+
+#define RGBLIGHT_LAYERS
+#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF
+#define RGBLIGHT_LIMIT_VAL 180
+#define RGBLIGHT_DEFAULT_MODE (RGBLIGHT_MODE_RAINBOW_SWIRL + 2)
+
+#define RGBLIGHT_SLEEP
+#define TAPPING_TOGGLE 2
+#define IOS_DEVICE_ENABLE
```
Comment (zvecr) [inline keyboards/quarkeys/z60/config.h:51]: Not a thing
```suggestion
```

File: keyboards/quarkeys/z60/config.h [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#define RGBLIGHT_EFFECT_RGB_TEST
+#define RGBLIGHT_EFFECT_ALTERNATING
+
+#define RGBLIGHT_LAYERS
+#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF
+#define RGBLIGHT_LIMIT_VAL 180
+#define RGBLIGHT_DEFAULT_MODE (RGBLIGHT_MODE_RAINBOW_SWIRL + 2)
+
+#define RGBLIGHT_SLEEP
+#define TAPPING_TOGGLE 2
```
Comment (zvecr) [inline keyboards/quarkeys/z60/config.h:50]:
```suggestion
```

File: keyboards/quarkeys/z60/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+layer_state_t layer_state_set_user(layer_state_t state) {
+    // Both layers will light up if both kb layers are active
+	//rgblight_set_layer_state(0, layer_state_cmp(state, 0));
+    rgblight_set_layer_state(1, layer_state_cmp(state, 1));
+    rgblight_set_layer_state(2, layer_state_cmp(state, 2));
+    rgblight_set_layer_state(3, layer_state_cmp(state, 3));
+    //rgblight_set_layer_state(4, layer_state_cmp(state, 4)); 
+    //rgblight_set_layer_state(5, layer_state_cmp(state, 5));
+    return state;
+}
```
Comment (zvecr) [inline keyboards/quarkeys/z60/keymaps/default/keymap.c:98]: We wouldnt want all the dead commented out code here

File: keyboards/quarkeys/z60/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+     * |------------+------+------+------+------+------|------+------+------+------+------+------+--------------|
+     * | SHIFT  |  \   |  Z   |  X   |  C   |  V   |  B   |  N   |  M   |  ,   |  .   |  /   |   Shift   |  Fn  |
+     * |--------+------+------+------+------+------+------+------+------+------+------+------+-----------+------|
+     * | Ctrl   |  Win   |  ALT   |                  SPACE                     |  ALT  |  Win  |  Manu |  Ctrl  |
+     * `--------+--------+--------+--------------------------------------------+-------+-------+-------+--------'     
+     */
+	[0] = LAYOUT(
+		KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL, 
+		LT(3, KC_TAB),    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+		LT(4, KC_CAPS),   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_HASH, KC_ENT,
```
Comment (zvecr) [inline keyboards/quarkeys/z60/keymaps/default/keymap.c:34]: `LT(4, KC_CAPS)` goes out of bounds

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by tommyamoszhao** [comment]
Need to change keyboard type directory and directory only, close this pull request.

**Pull request closed without merging**